core/server/data/schema/versioning.js

```diff
@@ -71,58 +71,9 @@ function setDatabaseVersion(options) {
         });
 }
 
-/**
```

Comment:
Removed some helper functions from versioning unit, because they are not needed anymore.
Knex-migrator auto detects which migrations needs to happen.